### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Tessyrich**
For this part of the project, I did the following;

- Setup the project with webpack
- I used every step in the project requirement.

### CODE REVIEW

File: modules/todoData.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const todos = [
```
Comment (gemmen29) [inline modules/todoData.js:1]: - [ ] Kindly follow the same names suggested for the task three keys (index, completed, description) as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Each%20task%20object,index%3A%20%5Bnumber%5D.).

File: modules/showTodo.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import todos from './todoData.js';
+
+const showTodo = () => {
+  const todoList = document.querySelector('.todo-list');
+  const todoApp = todos.map((todo) => `
```
Comment (gemmen29) [inline modules/showTodo.js:6]: - [ ] Kindly sort tasks based on their index values as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Tessyrich, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Tessyrich , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**